src/bin/rugs_metadata_server.rs

```diff
@@ -216,6 +225,44 @@ mod tests {
         Ok(pool)
     }
 
+    /// Test reading auth config secrets from files
+    #[tokio::test]
+    async fn config_secrets() -> Result<()> {
+        std::fs::write("./rugs_user_auth", USER_AUTH).expect("Could not write user secret to file");
```

Comment:
I think it would be good to add a dev dependency on `tempfile` or similar and write to a tempfile instead of this arbitrary file in the user's filesystem. (I actually don't know what the CWD of `cargo test` is.)